Document/0x05j-Testing-Resiliency-Against-Reverse-Engineering.md

```diff
@@ -1237,7 +1237,17 @@ The goal of device binding is to impede an attacker who tries to both copy an ap
 Before we describe the usable identifiers, let's quickly discuss how they can be used for binding. There are three methods that allow device binding:

```

Comment:
We now have more than three methods.

Reply:
Hi @TheDauntless,

I believe we still have three methods listed in the context. I have just elaborated and removed the scope of any ambiguities on one point that was already mentioned.

Reply:
My bad, didn't see the indent of the 2nd list :).